### PR TITLE
Feed UI update working branch [WIP]

### PR DESCRIPTION
**Pull request by pfrazee**
A collection of updates to the feed experience that we want to land all at once

- [x] https://github.com/bluesky-social/social-app/pull/1398
  - [ ] Ensure error-handling is still good on mergefeed
  - [x] Add temporary toggle to disable the mergefeed; this will soon be replaced with better overall feed config options (like making following a feed same as any)
- [x] https://github.com/bluesky-social/social-app/pull/1403
- [x] https://github.com/bluesky-social/social-app/pull/1439
- [x] https://github.com/bluesky-social/social-app/pull/1448
- [x] https://github.com/bluesky-social/social-app/pull/1455

# [Demo Video](https://www.youtube.com/watch?v=gwnEMqlMQxE)

### CODE REVIEW

File: src/view/com/posts/FeedItem.tsx [hunk 1]
```diff
@@ -179,7 +176,27 @@ export const FeedItem = observer(function FeedItemImpl({
...
+                <DesktopWebTextLink
+                  type="sm-bold"
+                  style={pal.textLight}
+                  lineHeight={1.2}
+                  numberOfLines={1}
+                  text={sanitizeDisplayName(source.displayName)}
+                  href={source.uri}
+                />
+              </Text>
+            </Link>
```
Comment (estrattonbailey) [inline src/view/com/posts/FeedItem.tsx:198]: What can we do to make this source text feel more like it applies to the following post(s) and not as a part of the top posts's UI? I think a little more whitespace on bottom, maybe align to left of avis? I liked the icon you had in there once.

Screen Shot 2023-09-15 at 2 03 47 PM

File: src/view/screens/Notifications.tsx [hunk 2]
```diff
@@ -88,9 +97,48 @@ export const NotificationsScreen = withAuthRequired(
...
+                paddingVertical: 12,
+              },
+            ]}>
+            <TextLink
+              type="title-lg"
+              href="/notifications"
+              style={{fontWeight: 'bold'}}
+              text={
+                <>
+                  Notifications{' '}
```
Comment (estrattonbailey) [inline src/view/screens/Notifications.tsx:120]: This text doesn't change in dark mode. Looks soooo good in light though 🥵 

Screen Shot 2023-09-15 at 2 25 35 PM

File: src/view/shell/bottom-bar/BottomBarWeb.tsx [hunk 3]
```diff
@@ -66,12 +65,11 @@ export const BottomBarWeb = observer(function BottomBarWebImpl() {
...
-          const Icon = isActive ? SatelliteDishIconSolid : SatelliteDishIcon
           return (
-            <Icon
-              size={25}
-              style={[styles.ctrlIcon, pal.text, styles.searchIcon]}
-              strokeWidth={1.8}
+            <HashtagIcon
+              size={22}
+              style={[styles.ctrlIcon, pal.text, styles.feedsIcon]}
+              strokeWidth={isActive ? 3 : 2.5}
```
Comment (estrattonbailey) [inline src/view/shell/bottom-bar/BottomBarWeb.tsx:72]: Not critical rn but this active state is pretty hard to see. Maybe we need a different indicator for this bottom bar in the near future 🤔

**Review by estrattonbailey — COMMENTED** [review]
Literally can't find anything wrong with it and it's a huge bump for usability imo 🚀

File: src/view/com/posts/FeedItem.tsx (hunk 1 above)
Comment (pfrazee) [reply src/view/com/posts/FeedItem.tsx:198]: Gonna sit on this one for a bit, haven't found anything I like yet

File: src/view/screens/Notifications.tsx (hunk 2 above)
Comment (pfrazee) [reply src/view/screens/Notifications.tsx:120]: Good catch, fixed

**Pull request merged**